Add a widget that shows the context info

jcoyne opened the pull request:
Fixes #223
<img width="1014" alt="Screenshot 2022-11-16 at 3 57 01 PM" src="https://user-images.githubusercontent.com/92044/202303366-79cb85cf-170a-4e72-9b52-b5162c379d79.png">

ggeisler commented:
Thanks for the screenshot. This doesn't say "Draft" so I'll just note:

- The buttons aren't styled as suggested in the ticket. 
- "Disable Session" is definitely not a button label we want to show end-users. I think my suggestion of "Don't show again" is fine, even if technically the user will eventually see it again (in another session or whatever).

jcoyne commented:
@ggeisler thanks for the feedback. I've fixed the text (translation address was wrong). I've updated the style to be btn-outline-primary for the button. However, when I added `style="background-color: #FFFFFF"` it causes problems with contrast as it interferes with the hover style for buttons provided by bootstrap.

ggeisler commented:
@jcoyne Okay, no problem. I did the mockup in Web inspector and it worked fine with hover, but maybe that wasn't completely realistic. We can go with the styling you have and I can look more closely at what we can do after it's merged.
